src/main/java/it/pagopa/ecommerce/commons/domain/v1/TransactionExpired.java

```diff
@@ -51,7 +52,7 @@ public Transaction applyEvent(Object event) {
             case TransactionRefundErrorEvent e ->
                     new TransactionWithRefundError(this.getTransactionAtPreviousState(), e);
```

Comment:
EXPIRED -> REFUND_ERROR should not be possible, only EXPIRED -> REFUND_REQUESTED

Reply:
Good catch, fixed with https://github.com/pagopa/pagopa-ecommerce-commons/pull/48/commits/5ebcb469b99080d12ddd1f34a4d203f7aa018f8b